src/pages/EventsCalendar/index.vue

```diff
@@ -1,14 +1,31 @@
 <template>
-  <events-calendar :events="events" :events-dates="eventsDates" />
+  <events-calendar
+    :events="events"
+    @selectDate="selectDate"
+    :events-dates="eventsDates"
+  />
 </template>
 
 <script setup lang="ts">
 import { Event } from 'src/utils';
 import { ref } from 'vue';
 import EventsCalendar from './EventsCalendar.layout.vue';
+import { mock_events, events_dates } from './mock';
+import { getDateFromTimestamptz } from './utils/parseTimestamptz';
 
 const events = ref<Event[]>([]);
-const eventsDates = ref<string[]>([]);
+const eventsDates = ref<string[]>(events_dates);
 
-// Add here the logic to get the events for the current user and save it in events and eventsDates variables
+const selectDate = (selectedDateString: string) => {
+  const selectedDate = new Date(selectedDateString);
+
+  events.value = mock_events.filter((event) => {
```

Comment:
please include the mock into a function that later can be substituted to a call to hasura

Reply:
like

getEvents = () => {
events=mock
}

Reply:
thanks for the suggestion!, I made the change, please check